src/auth/googleauth.ts

```diff
@@ -111,6 +111,8 @@ export class GoogleAuth {
    * @private
    */
   private checkIsGCE?: boolean = undefined;
+  useJWTAccessAlways?: boolean;
```

Comment:
Can we bike shed this name a little bit?  From a customer perspective, I'm not sure if I'd know what `useJWTAccessAlways` would mean.  Would it make more sense to use something like `useSelfSignedJWT`?

Reply:
@JustinBeckwith I believe this to be consistent with other language's implementation.

My recommendation was that we allow this as a class variable, but don't advertise it as an option on the options interface. The reason being that we want to be able to set this option in GAPIC libraries, but don't want to encourage customers to set the variable.

Reply:
okies - my feedback ain't blocking, and consistency is good